packages/react-label/src/components/Label/useLabel.tsx

```diff
@@ -24,7 +24,7 @@ export const useLabel = (props: LabelProps, ref: React.Ref<HTMLElement>): LabelS
     disabled,
     required: resolveShorthand(required === false ? null : required, {
       required: !!required,
-      defaultProps: { children: '*' },
+      defaultProps: { children: '*', 'aria-hidden': 'true' },
```

Comment:
Shouldn't this be just `true` (boolean)? Or is that not how this prop works?

Reply:
I believe it should be a string

Reply:
I think both will end up being rendered as `aria-hidden="true"` in HTML, but I also prefer the literal string, since it's clearer (at least to me, since `aria-hidden` is not a boolean prop, it takes one of two token string values, "true" or "false").